app/assets/javascripts/application/registration.js

```diff
@@ -0,0 +1,21 @@
+var registration = (function() {
+  var module = {};
+  var urlDescription = null;
+  var urlInput = null;
+
+  module.init = function() {
+    urlDescription = document.getElementById('url-description');
```

Comment:
Maybe it's a bit more consistent if we use jQuery for all DOM related calls

Reply:
I would not use jQuery if not necessary, maybe it brings a bit of inconsistency, but later all javascript should be updated according the pimped asset pipeline or webpack.